xarray/plot/utils.py

```diff
@@ -406,9 +406,12 @@ def _assert_valid_xy(darray, xy, name):
         raise ValueError(f"{name} must be one of None, '{valid_xy_str}'")
 
 
-def get_axis(figsize, size, aspect, ax):
-    import matplotlib as mpl
-    import matplotlib.pyplot as plt
+def get_axis(figsize, size, aspect, ax, **kwargs):
```

Comment:
each of these parameters should probably get a default (`None`?):
```suggestion
def get_axis(figsize=None, size=None, aspect=None, ax=None, **kwargs):
```
that means if we just want to set e.g. `ax`, we can use
```python
get_axis(ax=ax)
```